### ISSUE TITLE
Wrong Revit Geometry import by selection instance

### ISSUE THREAD

**Issue by redinkinc** [DynamoDS/DynamoRevit#1437]
## Dynamo version

1.2.0

## Revit version

2017 (german)

## Operating system

win10

## What did you do? 

Create a Revit Mass by Sweeping a profile along a segment of a circle.

## What did you expect to see?

the exact same geometry.

## What did you see instead?

![image](https://cloud.githubusercontent.com/assets/3741716/21592300/95df5484-d10c-11e6-846f-be5b6001e439.png)

**Comment by dimven** [comment]
Looks similar to https://github.com/DynamoDS/Dynamo/issues/3679

**Comment by redinkinc** [comment]
Yes, it is similar, only difference seems to be that no exceptions are thrown and I can use the geometry in any way. Just that there isn't it isn't the geometry I want to use. Is there any progress since?

**Comment by mjkkirschner** [comment]
@Racel @kronz has this been tracked in new tracking system?

**Comment by Racel** [comment]
@mjkkirschner - Not that I know of. Can you please file? And, is this still reproducible in 2.0?

**Comment by mjkkirschner** [comment]
https://forum.dynamobim.com/t/dynamo-and-sweep-issue/50400/2

**Comment by firas007noori** [comment]
Greeting dear all,

I am making a basic sweep in Revit, it made of 2 lines and then I filleted the corner to make a curved corner, and when I select the sweep to Dynamo, I get a circle instead of a curved corner!!
Any idea why?? and what is the solutions??
Many thanks to you in advance
![Capture2](https://user-images.githubusercontent.com/62472732/81435678-50269e80-9179-11ea-8629-9d88e3c78371.JPG)
![Capture](https://user-images.githubusercontent.com/62472732/81435723-5fa5e780-9179-11ea-9c0f-89d4e44befca.JPG)
